Error while launching the language server

I have followed the Development Setup,

```
pnpm install
pnpm run build
```

But I have encountered several errors:

```
extension\src\index.ts → extension/dist/extension/index.browser.js...
> node_modules/.pnpm/minimatch@3.1.2/node_modules/minimatch/minimatch.js:4:47: error: Plugin "rollup" returned a non-absolute path: path (set a namespace if this is not a file path)
4 │ var path = (function () { try { return require('path') } catch (e) {}}()) || {
╵                                                ~~~~~~

[!] (plugin esbuild) Error: Build failed with 1 error:
node_modules/.pnpm/minimatch@3.1.2/node_modules/minimatch/minimatch.js:4:47: error: Plugin "rollup" returned a non-absolute path: path (set a namespace if this is not a file path)
Error: Build failed with 1 error:
node_modules/.pnpm/minimatch@3.1.2/node_modules/minimatch/minimatch.js:4:47: error: Plugin "rollup" returned a non-absolute path: path (set a namespace if this is not a file path)
at failureErrorWithLog (C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:1475:15)
at C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:1133:28
at runOnEndCallbacks (C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:923:63)
at buildResponseToResult (C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:1131:7)
at C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:1240:14
at C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:611:9
at handleIncomingPacket (C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:708:9)
at Socket.readFromStdout (C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\esbuild@0.12.29\node_modules\esbuild\lib\main.js:578:7)
at Socket.emit (node:events:526:28)
at addChunk (node:internal/streams/readable:315:12)
```

And if I tried to start the language server, I get:

```
C:\Users\xxx\Downloads\workspace\grammarly\packages\grammarly-languageserver\dist\index.cjs.js:6
const fetch = require('node-fetch');
^

Error [ERR_REQUIRE_ESM]: require() of ES Module C:\Users\xxx\Downloads\workspace\grammarly\node_modules\.pnpm\node-fetch@3.2.4\node_modules\node-fetch\src\index.js from C:\Users\xxx\Downloads\workspace\_backup\grammarly\packages\grammarly-languageserver\dist\index.cjs.js not supported.
Instead change the require of index.js in C:\Users\xxx\Downloads\workspace\grammarly\packages\grammarly-languageserver\dist\index.cjs.js to a dynamic import() which is available in all CommonJS modules.
at Object.<anonymous> (C:\Users\xxx\Downloads\workspace\grammarly\packages\grammarly-languageserver\dist\index.cjs.js:6:15)
at Object.<anonymous> (C:\Users\xxx\Downloads\workspace\grammarly\packages\grammarly-languageserver\bin\server.js:3:16) {
code: ←[32m'ERR_REQUIRE_ESM'←[39m
```

Any idea are appreciated!

--- a/fixtures/readme.md
+++ b/fixtures/readme.md
@@ -8,7 +8,7 @@ also showed you how to correctly write the sentence.
 _Typical highlights works_ well when writing but they are a bit lmpty .
 Underlines **that are** blue indicate that Grammarly has spotted an unnecessarily wordy sentence. You’ll find suggestions that
 can possibly help you revise a wordy **sentence** in
-an effortless manner.
+an [effortless](https://example.com) manner.
 But wait...there’s more?
 
 Grammarly Premium can give you very helpful feedback on your writing. Passive voice can be fixed by Grammarly, and it can handle classical word-choice mistakes. It can also help with inconsistencies such as switching between Email, e-mail, and email or the U.S.A. and the USA.

--- a/packages/grammarly-languageserver/src/polyfill-fetch.ts
+++ b/packages/grammarly-languageserver/src/polyfill-fetch.ts
@@ -1,9 +1,10 @@
 // @ts-nocheck
-import fetch, { Request, Response, Headers } from 'node-fetch'
-
 if (typeof global.fetch === 'undefined') {
-  global.fetch = fetch
-  global.Request = Request
-  global.Response = Response
-  global.Headers = Headers
+  ;(async () => {
+    const { default: fetch, Request, Response, Headers } = await import('node-fetch')
+    global.fetch = fetch
+    global.Request = Request
+    global.Response = Response
+    global.Headers = Headers
+  })()
 }

